subsys/bluetooth/shell/bt.c

```diff
@@ -2308,6 +2308,7 @@ static const struct shell_cmd bt_commands[] = {
 	{ "id-delete", cmd_id_delete, "<id>" },
 	{ "id-show", cmd_id_show, HELP_NONE },
 	{ "id-select", cmd_id_select, "<id>" },
+	{ "ll-addr", cmd_ll_get_addr, "<random|public>"},
```

Comment:
Shouldn't this be next to `cmd_test_tx` and depend on `BT_CTLR`? This won't be available in Host-only builds.

Reply:
There's shell/ll.c which I commented about earlier. Seems natural that any "ll" shell commands would go there.

Reply:
Ah, I see the function implementation is there, but apparently we still have a unified table for the callbacks. Yes, in that case this definitely needs to be behind appropriate #if guards so that it's only included when there's an actual LL implementation part of the build/

Reply:
I have corrected this, also aligning with the discussion. Is the dependency in shell/ll.c correct too right it is done now, or should I remove any conditional compilation instructions from ll.c file? Referencing what @jhedberg says here I would assume that I can safetly remove #ifdef around cmd_ll_get_addr function definition now.

Reply:
Yes, ll.c will not get build if LL is not part of the build. So the c-file shouldn't need any #ifdefs

Reply:
It should be fixed right now.
@carlescufi @jhedberg - could you rereview?